docs/source/openmw-mods/convert_bump_mapped_mods.rst

```diff
@@ -6,23 +6,23 @@ Normal maps from Morrowind to OpenMW
     - `Normal Mapping in OpenMW`_
     - `Activating normal mapping shaders in OpenMW`_
     - `Normal mapping in Morrowind with Morrowind Code Patch`_
-    - `Normal mapping in Morrowind with MGE XE (PBR version)`_
+    - `Normal mapping in Morrowind with MGE XE`_
 - `Converting PeterBitt's Scamp Replacer`_ (Mod made for the MGE XE PBR prototype)
     - `Tutorial - MGE`_
 - `Converting Lougian's Hlaalu Bump mapped`_ (MCP's fake bump map function, part 1: *without* custom models)
     - `Tutorial - MCP, Part 1`_
-- `Converting Apel's Various Things – Sacks`_ (MCP's fake bump map function, part 2: *with* custom models)
+- `Converting Apel's Various Things - Sacks`_ (MCP's fake bump map function, part 2: *with* custom models)
     - `Tutorial - MCP, Part 2`_
 
 General introduction to normal map conversion
 ------------------------------------------------
 
 :Authors: Joakim (Lysol) Berg
-:Updated: 2016-11-09
+:Updated: 2016-11-11
 
-This page has general information and tutorials on how normal mapping works in OpenMW and how you can make mods using the old fake normal mapping technique (such as `Netch Bump mapped`_ and `Hlaalu Bump mapped`_ work in OpenMW. 
+This page has general information and tutorials on how normal mapping works in OpenMW and how you can make mods using the old fake normal mapping technique (such as `Netch Bump mapped`_ and `Hlaalu Bump mapped`_, and maybe the most (in)famous one to give shiny rocks in OpenMW, the mod `On the Rocks`_!, featured in MGSO and Morrowind Rebirth) work in OpenMW. 
 
-*Note:* The conversion made in part four of this tutorial require the use of the application NifSkope, which is a Windows-only application. Read part one of this tutorial to learn more.
+*Note:* The conversion made in the `Converting Apel's Various Things - Sacks`_-part of this tutorial require the use of the application NifSkope, which is a Windows-only application. Please inform us if you get that application to work on other OS:es via Wine or similar.
```

Comment:
@scrawl I see, thanks. I did notice that it seemed to be an open source application, but since there were no linux binaries available, I more or less assumed that it was not cross platform. I'll have this part changed then.

@MiroslavR Ok, so basically you can't get the later versions compiled (at the moment at least) for Linux, but you can with older? Would that be the correct info?